03LessonThree/homework/git-scm.html

```diff
@@ -7,21 +7,21 @@
 </head>
 
 <body>
-    <div id="header">
-        <div id="nav">
+    <header>
+        <nav>
             <span><a href="#">Home</a></span>
             <span><a href="#">How does it work</a></span>
             <span><a href="#">Contact Us</a></span>
         </div>
     </div>

```

Comment:
Line 15 needs to be the closing `</nav>` tag and line 16 needs to be the closing `</header>` tag.